python/ray/_private/ray_constants.py

```diff
@@ -581,3 +581,6 @@ def gcs_actor_scheduling_enabled():
 )
 
 RAY_GC_MIN_COLLECT_INTERVAL = env_float("RAY_GC_MIN_COLLECT_INTERVAL_S", 5)
+
+WARN_BLOCKING_GET_INSIDE_ASYNC_ENV_VAR = "WARN_BLOCKING_GET_INSIDE_ASYNC"
+DEFAULT_WARN_BLOCKING_GET_INSIDE_ASYNC = True
```

Comment:
what's the point of this var if it's hard coded to true?

Reply:
removed and just inlined `env_bool(...True)`